projects/packages/compat/composer.json

```diff
@@ -29,11 +29,12 @@
 	"extra": {
 		"autotagger": true,
 		"mirror-repo": "Automattic/jetpack-compat",
+		"textdomain": "jetpack-compat",
```

Comment:
If we add a textdomain here, let's also add some linting to ensure it will be respected. Something like this:
https://github.com/Automattic/jetpack/blob/73c2eb27b471c2904494a24cc3f8f066f2e030cf/projects/packages/connection/.phpcs.dir.xml#L4-L22

Reply:
Good call. Added!